Magento2/Sniffs/Less/ColourDefinitionSniff.php

```diff
@@ -42,7 +42,7 @@ public function process(File $phpcsFile, $stackPtr)
 
         $variablePtr = $phpcsFile->findPrevious(T_ASPERAND, $stackPtr);
         if ((false === $variablePtr) || ($tokens[$stackPtr]['line'] !== $tokens[$variablePtr]['line'])) {
-            $phpcsFile->addError('Hexadecimal value should be used for variable', $stackPtr, 'NotInVariable');
+            $phpcsFile->addError('Hexadecimal value should be get from a variable', $stackPtr, 'NotInVariable');
```

Comment:
This can be more clear error description in my opinion:
```suggestion
            $phpcsFile->addError('A varialbe should be used for a CSS colour', $stackPtr, 'NotInVariable');
```

Reply:
@sivaschenko done